cli/cmd/install.go

```diff
@@ -690,6 +699,19 @@ func (options *installOptions) validate() error {
 	return nil
 }
 
+func validateEndpointSlicesFeature(enableEndpointSlices bool) error {
+	if !enableEndpointSlices {
```

Comment:
generally I prefer to let the calling code control things like directly rather than passing instructions into the function.  In this case, I'd remove the enableEndpointsSlices argument and instead call this function like:


```
if enableEndpointSlices {
  if err := validateEndpointsSlicesFeature(); err != nil {
    return ....
  }
}
```

Reply:
I can see why that's a better choice :) Thanks for signalling this, I've changed the code, squashed the commits and pushed.